README.md

```diff
@@ -26,34 +35,94 @@ Forklift provides tools to create Foreman/Katello environments for development,
 ### Requirements
 
 * Vagrant - 2.2+ - Both the VirtualBox and Libvirt providers are tested
-* Ansible - 2.9+
+* Ansible - 2.12+
 * [Vagrant Libvirt provider plugin](https://github.com/vagrant-libvirt/vagrant-libvirt) (if using Libvirt)
 * Virtualization enabled in BIOS
 
 See [Installing Vagrant](docs/vagrant.md) for installation instructions.
 
 ### Quickstart
 
-This will walk through the simplest path of spinning up a production test environment of a bleeding edge nightly installation assuming Vagrant and Libvirt are installed and configured.
+The quickstart guide covers the basic steps to install Forklift so that you can quickly spin up a production test environment of the latest bleeding edge nightly Foreman or Katello build.
 
-```
+It assumes you have already Vagrant and libvirt installed, as well as the vagrant-libvirt and vagrant-hostmanager plugins. You also need Ansible, although if you don't have it or aren't sure if your version is compatible, you can simply create and activate a Python virtual environment and then `pip install ansible-galaxy`
+
+Start by cloning the Forklift repository:
+
+```sh
 git clone https://github.com/theforeman/forklift.git
 cd forklift
+```
+
+Then install the Ansible collections which are needed by Forklift:
+
+```sh
 ansible-galaxy collection install -r requirements.yml
-vagrant up centos7-foreman-nightly
 ```
 
-The same can be quickly done for a development environment where GITHUB_NICK is your GitHub username:
+When this is complete, simply follow the next section of this README to try out the latest nightly Foreman build using Forklift.
+
+### Foreman Nightly Box
```

Comment:
This should mostly be covered in https://github.com/theforeman/forklift/blob/master/docs/development.md already. https://theforeman.github.io/forklift/development/ should be the rendered version of that.